fatrace.c

```diff
@@ -316,7 +317,7 @@ static void
 print_event (const struct fanotify_event_metadata *data,
              const struct timeval *event_time)
 {
-    const char* problems[7];
+    const char* problems[14];
```

Comment:
Yah -- see my previous comment, please no magic numbers and arbitrary static sizes in C. It's going to bite you.

Reply:
See previous comment. I can of course do it dynamically if you want, but that is also unnecessary complexity. I added some comments and checks in #51. While the code is correct, I understand your concern; it'd be easy to inadvertently add an `add_problem` call and forget to increase the limit. Perhaps we can just use a higher constant for a safety margin? I increased it to 1000 in #51.